tools/cli/commands.js

```diff
@@ -724,12 +742,65 @@ main.registerCommand({
     return 0;
   }
 
-  var appPathAsEntered;
-  if (options.args.length === 1) {
-    appPathAsEntered = options.args[0];
-  } else {
-    throw new main.ShowUsage;
+  const setup = async () => {
+    const ask = createPrompt();
+
+    let appPathAsEntered, skeleton;
+    if(options.args.length === 1) {
+      appPathAsEntered = options.args[0];
+    }
+    else {
+      appPathAsEntered = await ask(`What is the name/path of your ${yellow`app`}? `);
+    }
+
+    const skeletonExplicitOption = AVAILABLE_SKELETONS.find(skeleton =>
+      !!options[skeleton]);
+
+    if(skeletonExplicitOption) {
```

Comment:
If a name is provided as an arg, the interactivity should be disabled and it use the default skeleton if non was provided. This is needed to avoid breaking scripts that use `meteor create`.

Reply:
@zodern Yes, that would be one of cases where we wouldn't need to make prompts. 
So, in that case, we only bring up the prompts when app name is not provided?

Reply:
> So, in that case, we only bring up the prompts when app name is not provided?

Yes.

Showing a message like you suggest when using the default skeleton makes sense.

Reply:
@zodern made some changes